sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -1627,3 +1627,108 @@ def sdc_pandas_dataframe_groupby_impl(self, by=None, axis=0, level=None, as_inde
         return init_dataframe_groupby(self, column_id, grouped, sort)
 
     return sdc_pandas_dataframe_groupby_impl
+
+
+def sdc_pandas_dataframe_reset_index_codegen(df_type, all_params, columns, result_name, func_line):
+    joined = ', '.join(all_params)
+    func_lines = [f'def _df_reset_index_impl({joined}):']
+    df = all_params[0]
+    func_lines += func_line
+    for i, c in enumerate(columns):
+        result_c = f'result_{c}'
+        func_lines += [
+            f'  result_{c} = get_dataframe_data({df}, {i})'
```

Comment:
Please, use column id instead it's name to avoid fails when column names are with spaces